docs/spec/slashing/state-machine.md

```diff
@@ -0,0 +1,174 @@
+# State Machine Interaction Overview
+
+## Conceptual overview
+
+### States
+
+At any given time, there are any number of validator candidates registered in the state machine.
+Each block, the top `n` candidates who are not jailed become *bonded*, meaning that they may propose and vote on blocks.
+Validators who are *bonded* are *at stake*, meaning that part or all of their stake is at risk if they commit a protocol fault.
+
+### Slashing period
+
+In order to mitigate the impact of initially likely categories of non-malicious protocol faults, the Cosmos Hub implements for each validator
+a *slashing period*, in which the amount by which a validator can be slashed is capped at the punishment for the worst violation. For example,
+if you misconfigure your HSM and double-sign a bunch of old blocks, you'll only be punished for the first double-sign (and then immediately jailed,
```

Comment:
is this a realistic example? (fine if so) I just don't want to scare people into thinking if they slip up somehow in configuration they get messed - we should make hsm configuration pretty robust

Reply:
I think it is a realistic example, that's why the slashing period exists.

Agreed that we want to develop secure HSM software though!